src/__tests__/ParseSchema-test.js

```diff
@@ -69,6 +83,30 @@ describe('ParseSchema', () => {
     done();
   });
 
+  it('can create schema fields required and default values', (done) => {
```

Comment:
since this doesn't require any async, you could remove the done from the func declaration and remove the end done().